lumigator/frontend/src/router/index.ts

```diff
@@ -1,25 +1,24 @@
 import {
   createRouter,
   createWebHistory,
+  type RouteRecordRaw,
 } from 'vue-router';
 
 const LDatasetes = () => import('@/components/pages/LDatasets.vue')
 const LExperiments = () => import('@/components/pages/LExperiments.vue')
 
-export const routes = [
+export const routes: Array<RouteRecordRaw & {icon?: string}> = [
 	{
 		path: '/datasets',
 		name: 'datasets',
-		disabled: false,
 		component: LDatasetes,
-		icon: 'pi pi-dataset'
+		icon: 'pi pi-dataset',
 	},
 	{
 		path: '/experiments',
 		name: 'experiments',
-		disabled: false,
 		component: LExperiments,
-		icon: 'pi pi-experiments'
+		icon: 'pi pi-experiments',
```

Comment:
`disabled` is not a valid vue-router option, it didn't seem to be used explicitly elsewhere in the code either

ideally we should remove the `icon` from there aswell, but thats being used through `slotProps`, there's an opportunity to refactor this in the future to separate vue-router's config from that of the side menu